editor/help.md

```diff
@@ -0,0 +1 @@
+***mcqlugin***
```

Comment:
@sivashanmugam-funtoot Please fix the help.md content to reflect the correct plugin